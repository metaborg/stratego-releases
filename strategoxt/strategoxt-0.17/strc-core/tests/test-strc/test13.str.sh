#!/bin/sh

${STRC} -i test13.str -la stratego-lib -O 0|| exit 1 

rm -f test13.str.actual
./test13 > test13.str.actual 2>&1

diff test13.str.expected test13.str.actual || (echo "Result not expected; test failed"; exit 1)
