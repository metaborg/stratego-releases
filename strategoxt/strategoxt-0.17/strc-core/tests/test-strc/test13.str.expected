1
identity crisis: rewriting failed, trace:
	main_0_0
	foo_0_0
[ identity crisis | critical ] Internal error: with clause failed unexpectedly in rule 'foo'
           0
